e2e/e2e_test.go

```diff
@@ -559,7 +561,7 @@ openAPI:
           required: true
           isSet: true
  `,
-			errMsg: `Package uninitialized`,
+			errMsg: `completed`,
```

Comment:
reading the name field in the above two test cases suggests the test should fail with some error message ?

Reply:
Yeah, the name and the expected error is a bit misleading here. I have updated the PR to just expect no error in the last two test cases.